app/helpers/goals_helper.rb

```diff
@@ -0,0 +1,2 @@
+module GoalsHelper
+end
```

Comment:
please remove all helper files in this folder if not needed. We'll add/keep helper files only when necessary.